peach/src/main/java/com/hartwig/hmftools/peach/haplotype/Haplotype.java

```diff
@@ -0,0 +1,5 @@
+package com.hartwig.hmftools.peach.haplotype;
+
+public interface Haplotype
```

Comment:
This interface feels strange. Both implementations (`DefaultHaplotype` and `NonDefaultHaplotype`) basically have the same properties (`name`, `events` and `isWildType`), the only difference is interpretation of the events. 

I would move at least `name` and `isWildType` to the interface. That way you don't need to cast to access these fields.

Reply:
I have moved name and isWildType to the interface